plugin/testing_write.go

```diff
@@ -48,6 +48,17 @@ type SafeMigrations struct {
 	MovePKToCQOnly         bool
 }
 
+// Migrations defines which migrations should be skipped completely
+type Migrations struct {
+	AddColumn              bool
+	AddColumnNotNull       bool
+	RemoveColumn           bool
+	RemoveColumnNotNull    bool
+	ChangeColumn           bool
```

Comment:
```suggestion
```

Aren't all migrations support these? Maybe only allow the new ones?

Reply:
My reasoning was that new destinations might not support those migrations and so might as well expose them for future usage.

Reply:
I think the baseline should be to support these basic changes in all destinations. I'd only open this for configuration if we run into a destination that really needs them

Reply:
fixed in https://github.com/cloudquery/plugin-sdk/pull/1560/commits/b774c094ad9d394062bdffca8afec5dc7d623cf0